sql/table_test.go

```diff
@@ -168,7 +175,9 @@ func TestMakeTableDescIndexes(t *testing.T) {
 		}
 		schema, err := makeTableDesc(stmt[0].(*parser.CreateTable))
 		if err != nil {
```

Comment:
I think we should just have a single separate test case that validates that the primary key was specified, rather than polluting this test, and ditto for the above